tools/fiptool/fiptool.c

```diff
@@ -20,8 +20,8 @@
 
 #include <openssl/sha.h>
 
+#include <firmware_image_package.h>
```

Comment:
Ok - thinking about this a bit more, that seemed to be the intention of how these includes were laid out. So let's keep the groupings for the moment.

And yes agree that if we're grouping then need to have a blank line.